CHANGELOG.md

```diff
@@ -7,3 +7,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Added
 
 - Initial release
+
+## [0.0.2] - 20 Jun 2024
+
+### Added
+
+- Added ability to process gifs
+- Included cheering gif
```

Comment:
Changelogs should have the latest version first, see https://keepachangelog.com/en/1.1.0/